src/components/Dashboard/FilingHistoryList/CompletedDissolution.vue

```diff
@@ -49,6 +49,13 @@ export default class CompletedDissolution extends Mixins(DateMixin) {
 
   /** The dissolution date to display. */
   get dissolutionDate (): string {
+    // try to get dissolution date from filing
+    const dd = this.filing?.dissolutionDate || this.filing?.dissolution?.dissolutionDate
+    if (dd) {
+      const dissolutionDate = this.yyyyMmDdToDate(dd)
+      return (this.dateToPacificDate(dissolutionDate, true) || 'Unknown')
+    }
+    // fall back to using effective date
     return (this.dateToPacificDate(this.filing?.effectiveDate, true) || 'Unknown')
   }
```

Comment:
At the moment, the dissolution date is not present in the filing data. I'm taking a guess above at where it might be.

So this filing has 3 dates:
- submitted date (when it was filed by the user)
- effective date (when it was processed by the filer)
- dissolution date (the date of the event in the filing)

Reply:
yeah we will need to figure out where to populate that.  Right now the `data` block coming back for a filing via `/api/v2/businesses/FM0278254/filings` endpoint is directly from the meta data block.  One option is to add the dissolution date in there.  And if I do it asap, I'm still good for the data loading to Prod.  I'll bring this up in grooming tomorrow.

Reply:
Note that there's a difference between `GET https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v2/businesses/FM0278254/filings`:

![image](https://user-images.githubusercontent.com/10002889/194207601-79c010fb-d355-413e-8f61-309354f514e0.png)

And `GET https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v2/businesses/FM0278254/filings/137409`:

![image](https://user-images.githubusercontent.com/10002889/194207706-bf8a78bd-dde0-4bab-aec7-6a0bb3fa30a8.png)

Reply:
I'm going to assume the dissolution date will be at `filing.data.dissolution.dissolutionDate`.